packages/http/src/validator/validators/utils.ts

```diff
@@ -3,23 +3,20 @@ import { DiagnosticSeverity } from '@stoplight/types';
 import { getSemigroup, NonEmptyArray, fromArray, map } from 'fp-ts/lib/NonEmptyArray';
 import { getValidation } from 'fp-ts/lib/Either';
 import * as O from 'fp-ts/lib/Option';
+import { Do } from 'fp-ts-contrib/lib/Do';
 import { pipe } from 'fp-ts/lib/pipeable';
 import { sequenceT } from 'fp-ts/lib/Apply';
-import * as Ajv from 'ajv';
+import type { ErrorObject } from 'ajv';
```

Comment:
This makes `ajv` a devDependency if you wanted to do that. As far as I checked, there's no surface-visible `ajv` usage left, it's only used in tests.